src/integer_mod_q/modulus_polynomial_ring_zq/from.rs

```diff
@@ -11,16 +11,155 @@
 //! The explicit functions contain the documentation.
 
 use super::ModulusPolynomialRingZq;
-use crate::{error::MathError, integer_mod_q::PolyOverZq, macros::for_others::implement_for_owned};
+use crate::{
+    error::MathError,
+    integer::{PolyOverZ, Z},
+    integer_mod_q::{Modulus, PolyOverZq, Zq},
+    macros::for_others::implement_for_owned,
+};
 use flint_sys::fq::fq_ctx_init_modulus;
 use std::{ffi::CString, mem::MaybeUninit, rc::Rc, str::FromStr};
 
+impl From<&Zq> for ModulusPolynomialRingZq {
+    /// Creates a constant [`ModulusPolynomialRingZq`], i.e. the polynomial `x mod q`,
+    /// where `x` is the value of the given [`Zq`] value and `q` its modulus.
+    ///
+    /// Parameters:
+    /// - `value`: the constant value the polynomial will have.
+    ///   
+    /// Returns a new constant [`ModulusPolynomialRingZq`] with the specified
+    /// `value` and `modulus` of the [`Zq`] value.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::{integer_mod_q::*, traits::*};
+    ///
+    /// let mod_poly = ModulusPolynomialRingZq::from(&Zq::from((2, 10)));
+    ///
+    /// let poly_cmp = ModulusPolynomialRingZq::from((2, 10));
+    /// assert_eq!(mod_poly, poly_cmp);
+    /// assert_eq!(mod_poly.get_degree(), 0);
```

Comment:
```suggestion
    /// use qfall_math::{integer_mod_q::*, traits::*};
    ///
    /// let zq = Zq::from((2, 10));
    ///
    /// let mod_poly = ModulusPolynomialRingZq::from(&zq);
```
Just a suggestion. Looks more concise to me as an example on how to use the function.